crates/shared/Cargo.toml

```diff
@@ -58,4 +58,5 @@ web3 = { version = "0.18", default-features = false }
 [dev-dependencies]
 flate2 = "1.0"
 regex = "1.5"
+sqlx = { version = "0.6", default-features = false, features = ["runtime-tokio-native-tls"] }
```

Comment:
Like Nic in another PR I had to include this so the shared crate can run e2e tests.